src/system/applications/combat/combat-tracker.ts

```diff
@@ -0,0 +1,156 @@
+import CosmereCombatant from './combatant';
+
+/**
+ * Overrides default tracker template to implement slow/fast buckets and activation button.
+ */
+export default class CosmereCombatTracker extends CombatTracker {
+    // eslint-disable-next-line @typescript-eslint/class-literal-property-style
```

Comment:
As a good habit, when I disable an eslint rule I add a note comment (`// NOTE: ...`) describing why that rule had to be disabled. This is useful for when other people look at the code (or yourself after a long while away) and forces you to really justify needing to disable eslint rules. 
Disabling eslint rules should be generally discouraged, but is sometimes needed